lib/presentation/widgets/registration_form.dart

```diff
@@ -57,21 +59,9 @@ class _MyLogFormWidgetState extends State<RegisterForm> {
           spacer,
           CustomButton(
             onPressed: () async {
-              try {
-                final credential = await FirebaseAuth.instance.createUserWithEmailAndPassword(
-                  email: _email.text,
-                  password: _pass.text,
-                );
-                print('New creation User');
-              } on FirebaseAuthException catch (e) {
-                if (e.code == 'weak-password') {
-                  print('The password provided is too weak.');
-                } else if (e.code == 'email-already-in-use') {
-                  print('The account already exists for that email.');
-                }
-              } catch (e) {
-                print(e);
-              }
+              context.read<RegistrationBloc>().add(CreateUserEvent(_pass.text, _email.text));
```

Comment:
there is always login first, password second

Reply:
ok